x-pack/filebeat/filebeat.reference.yml

```diff
@@ -453,6 +453,14 @@ filebeat.modules:
 - module: misp
   threat:
     enabled: true
+    # API key to access MISP
+    #var.api_key
+
+    # Array object in MISP response
+    #var.json_objects_array
+
+    # URI of the MISP REST API
```

Comment:
A URI is a superset of a URL. Shouldn't the comment reference the URL?

Reply:
Yep I think so! Thanks!